pipelines/unified_quickstart/alt-2_calculate_avg_gpa_gcp.wdl

```diff
@@ -0,0 +1,49 @@
+version 1.0
+
+workflow CalculateStudentGPA {
+    meta {
+        description: "Calculate the GPA (average) for a student."
+    }
+
+    input {
+        Float   math_score
+        Float   language_score
+        Float   science_score
+        Int     num_scores
+    }
+
+    call CalculateAverage {
+        input:
+            math_score      = math_score,
+            language_score  = language_score,
+            science_score   = science_score,
+            num_scores      = num_scores
+    }
+
+    output {
+        Float   gpa = CalculateAverage.average
+    }
+}
+
+task CalculateAverage {
+    input {
+        Float   math_score
+        Float   language_score
+        Float   science_score
+        Int     num_scores
+    }
+
+    command <<<
+        python -c "print((~{math_score} + ~{language_score} + ~{science_score})/~{num_scores})"
+
+    >>>
+    
+
+    runtime {
+        docker: "us.gcr.io/broad-dsp-gcr-public/base/python:3.12-debian"
+    }
```

Comment:
Despite the `-public` designation, I can't pull this image locally. Are we sure that users with no Broad affiliation can actually use this image seamlessly?

Granting workshop users access to an image seems like an unnecessary complexity compared to `python:slim`.
```
anichols@wm334-2ce ~> docker pull us.gcr.io/broad-dsp-gcr-public/base/python:3.12-debian
Error response from daemon: Head "https://us.gcr.io/v2/broad-dsp-gcr-public/base/python/manifests/3.12-debian": unauthorized: authentication failed
```

Reply:
there's no special access, it's just that GCP Artifact Registry requires setting up a command-line helper first, I believe: https://cloud.google.com/artifact-registry/docs/docker/pushing-and-pulling#cred-helper

Reply:
But I thought the idea is to share the Azure registry image not GCR? Or is it for GCP users? In that case, just have them set up this cred helper, I think.

Reply:
Just closing the loop here as well...will use `python:slim`

Reply:
And FWIW - the intention was to use GCR image for GCP workshops, and ACR image (referenced in a separate file in this PR) for Azure workshops.